lab-michael/greeting.js

```diff
@@ -0,0 +1,9 @@
+var myObject  = {};
+
+myObject.greet = function greet(name) {
```

Comment:
Given that this is just the singular greet function, I would suggest just assigning it directly to `module.exports`, and not worry about creating an object literal, assigning a method, and then exporting the obj.